integration-tests/actions/ocr2vrf_actions/ocr2vrf_steps.go

```diff
@@ -272,14 +273,14 @@ func RequestRandomnessFulfillmentAndWaitForFulfilment(
 }
 
 func getRequestId(t *testing.T, consumer contracts.VRFBeaconConsumer, receipt *types.Receipt, confirmationDelay *big.Int) *big.Int {
-	periodBlocks, err := consumer.IBeaconPeriodBlocks(nil)
+	periodBlocks, err := consumer.IBeaconPeriodBlocks(context.Background())
```

Comment:
These can be tied to the test:
```suggestion
	ctx := testutils.Context(t)
	periodBlocks, err := consumer.IBeaconPeriodBlocks(t)
```